scene/3d/visual_instance.cpp

```diff
@@ -207,6 +229,8 @@ VisualInstance::VisualInstance() {
 	instance = RID_PRIME(VisualServer::get_singleton()->instance_create());
 	VisualServer::get_singleton()->instance_attach_object_instance_id(instance, get_instance_id());
 	layers = 1;
+	sorting_use_aabb_center = true;
+	sorting_offset = 0;
```

Comment:
More of a nitpick in 3.x, but it's preferred to define the default values in the header instead of the constructor. This should definitely be the case in the `master` version, if there is one.

Reply:
Yeah for 3.x I let this go through as this is actually more consistent, since we didn't port most of the codebase.